### ISSUE TITLE
JSON files not ignored

### ISSUE THREAD

**Issue by miguelmota** [google/ts-style#227]
Example

`tslint.json`

```json
{
  "extends": "gts/tslint.json",
  "exclude": [
    "*.json",
    "**/*.json",
    "src/*.json",
    "src/datasources/*.json",
    "src/datasources/db.datasource.json"
  ]
}
```

Command

```
gts fix
```

Files modified

```
db.datasource.json
```

Contents

```
{
  "name": "",
  "connector": "",
  "host": "",
  "port":,
  "user": "",
  "password": "",
  "database": ""
}; <-- semicolon should not be inserted!
```

gts fix adds a semicolon to json making it invalid json

**Comment by ofrobots** [comment]
Duplicate of #172. We have a fix implemented in #220 but that is await a tslint release. Until then you can [this workaround](https://github.com/google/ts-style/issues/172#issuecomment-433321584).

**Comment by Eksapsy** [comment]
By the way I resolved this issue, by implementing this easy workaround, since [the workaround](https://github.com/google/ts-style/issues/172#issuecomment-433321584) didn't work for me.
**package.json**
```json
"scripts": {
    "fix": "gts fix ./src/**/*.ts"
}
```
